libraries/Microsoft.Bot.Builder.Dialogs/DialogManager.cs

```diff
@@ -368,5 +371,33 @@ private async Task<DialogTurnResult> HandleBotOnTurnAsync(DialogContext dc, Canc
 
             return turnResult;
         }
+
+        /// <summary>
+        /// Helper to determine if we should send an EoC to the parent or not.
+        /// </summary>
+        private bool SendEoCToParent(ITurnContext context, DialogTurnResult turnResult)
```

Comment:
nit: maybe a more descriptive method name. I kinda expect a method that starts "send..." to be sending something. How about "ShouldSendEoCToParent" ???

(I know this is internal and it avoids a stupidly long name but I'n not a fan of the abbreviation EoC either.)

Reply:
Done